src/i18n/en.json

```diff
@@ -618,6 +618,17 @@
       "LATEST": "Last Uploaded",
       "MOST_VIEWED": "Most Viewed"
     },
+    "STATS_BAR": {
+      "totalIdentifiers": "Identifiers",
+      "totalObservations": "Observations",
+      "totalTaxa": "Taxa",
+      "totalUploaders": "Uploaders"
```

Comment:
key titles should always be in upper case, I missed this earlier

Reply:
changed to upper case